happi/tests/test_client.py

```diff
@@ -82,6 +82,7 @@ def test_create_valve(happi_client, valve_info):
     assert device.beamline == valve_info['beamline']
     # Save
     device.save()
+    # seeing some format differences between entries in testing.json
```

Comment:
Still need this comment?

Reply:
No and my bad forgot I should be using the logger for messages like these - then won't have to remember to delete.